met-web/src/services/widgetService/index.tsx

```diff
@@ -73,3 +74,22 @@ export const removeWidget = async (engagement_id: number, widget_id: number): Pr
         return Promise.reject(err);
     }
 };
+
+export const sortWidgets = async (engagement_id: number, sort_index: number): Promise<Widget> => {
+    try {
+        const url = replaceAllInURL({
+            URL: Endpoints.Widgets.SORT,
+            params: {
+                engagement_id: String(engagement_id),
+                sort_index: String(sort_index),
```

Comment:
It looks to me that its missing the widget id that is being sorted?!

Reply:
its not in the back end may need some updates in the sorting method

Reply:
resolved with the widget index